engines/toon/movie.cpp

```diff
@@ -103,6 +106,7 @@ void Movie::playVideo(bool isFirstIntroVideo) {
 	while (!_vm->shouldQuit() && !_decoder->endOfVideo()) {
 		if (_decoder->needsUpdate()) {
 			const Graphics::Surface *frame = _decoder->decodeNextFrame();
+			byte unused = 0;
```

Comment:
maybe use of `bool` would be better?

Reply:
this variable should later get assigned with unused value in range [1, 256] afterwards (color key) - 0 means it failed to find unused value (and already can't be unused, as it used for text border)